documentation/modules/proc-sending-messages.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// assembly-evaluation.adoc
+
+[id='proc-using-amq-streams-{context}']
+
+= Sending and receiving messages from a topic
+
+Test your {ProductName} installation by sending and receiving messages outside the cluster from `my-topic`.
+
+Use a terminal to run a Kafka producer and consumer on a local machine.
+
+.Prerequisites
+
+* {ProductName} is installed on the Kubernetes cluster
+* Zookeeper and Kafka are running
+
+.Procedure
+
+. Download the latest Kafka binaries and install Kafka on your local machine.
++
+{ApacheKafkaDownload}
+
+. Find the port of the bootstrap service:
++
+[source, shell, subs=+quotes, options="nowrap"]
+----
+kubectl get service my-cluster-kafka-external-bootstrap -n my-kafka-project -o=jsonpath='{.spec.ports[0].nodePort}{"\n"}'
+----
+
+. Find the IP address of the Minikube node:
++
+[source, shell, subs=+quotes, options="nowrap"]
+----
+kubectl get nodes --output=jsonpath='{range .items[*]}{.status.addresses[?(@.type=="InternalIP")].address}{"\n"}{end}'
+----
+
+. Open a terminal and start the Kafka console producer with the topic `my-topic` and the authentication properties for TLS:
++
+[source,shell,subs=+quotes]
+----
+bin/kafka-console-producer.sh --broker-list _<node-address>_:_<node-port>_ --topic my-topic
+----
+
+. Type your message into the console where the producer is running.
+
+. Press Enter to send the message.
+
+. Open a new terminal tab or window and start the consumer to receive the messages:
++
+[source,shell,subs=+quotes]
+----
+bin/kafka-console-consumer.sh --bootstrap-server _<node-address>_:_<node-port>_ --topic my-topic --from-beginning
+----
+
+. Confirm that you see the incoming messages in the consumer console.
+
+. Press Crtl+C to exit the Kafka console consumer.
```

Comment:
The need to be told to exit the consumer, but not the producer?

Reply:
...producer and consumer...